test/verify/check-bots-api

```diff
@@ -112,6 +113,19 @@ class TestImageCustomize(unittest.TestCase):
                 subprocess.check_call(["bots/image-customize", "--verbose",
                                        "--run-command", "false", img])
 
+    def testResize(self):
+        dest = tempfile.mkdtemp(dir=".")
+        self.addCleanup(shutil.rmtree, dest)
+        img = os.path.join(dest, os.environ["TEST_OS"])
+
+        with testvm.Timeout(seconds=300, error_message="Timed out waiting for image-customize"):
+            subprocess.check_call(["bots/image-customize", "--verbose",
+                                   "--resize", "20G", img])
+
+            r = subprocess.run(["qemu-img", "info", "--output=json", img], capture_output=True, encoding="utf-8", check=True)
```

Comment:
This part should be outdented, not part of the Timeout.

Reply:
Clearly. Done. Thanks!